RecoTracker/LSTCore/src/alpaka/PixelTriplet.h

```diff
@@ -641,10 +642,9 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
         float diffz1 = alpaka::math::abs(acc, solz1 - zsi) * 100;
         float diffz2 = alpaka::math::abs(acc, solz2 - zsi) * 100;
         diffz = alpaka::math::min(acc, diffz1, diffz2);
+        residual = diffz;
```

Comment:
It is a bit weird that this line and the Line 628 are added into this PR, it seems pure pT3 rzchi2 work.

Reply:
Thank you, @YonsiG. Seems like I might have messed something up. I'll fix it.

Reply:
Actually, if you look at #40, those changes were made for pT3 rzchi2. Either way, those changes shouldn't affect anything.

Reply:
Yes, those are for pT3 rzchi2. The order doesn't affect much as you said.